components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -86,7 +77,8 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
       switch (error.code) {
         case ErrorCodes.CONTEXT_PARSE_ERROR:
           statusMessage = <div className="text-center">
-            <p className="text-base text-red-500">Unrecognized context: '{contextUrl}'</p>
+            <p className="text-base text-red-500">Problem with context: '{contextUrl}'</p>
+            <p className="text-base text-red-500">{error.message}</p>
```

Comment:
This is to clarify context-related problems that are not really "parsing errors", e.g. when you try to manually start a prebuild but there was already one in progress (https://github.com/gitpod-io/gitpod/issues/3687):

<img width="962" alt="Screenshot 2021-04-01 at 06 48 38" src="https://user-images.githubusercontent.com/599268/113245070-818aea00-92b6-11eb-808b-9bd3e483969c.png">

However, this makes error messages very long and unpleasant. It also makes actual "parsing errors" very redundant:

<img width="854" alt="Screenshot 2021-04-01 at 06 48 03" src="https://user-images.githubusercontent.com/599268/113245171-a5e6c680-92b6-11eb-9d71-f82493c39aba.png">

Not sure what the best solution is here.

Reply:
As discussed on Slack, we're now always showing the actual error message.

For context errors, we don't add the additional static error text seen above.